templates/files/conf/k8s-addons

```diff
@@ -121,19 +121,10 @@ done
 # delete calico-kube-controllers
 $KUBECTL -n kube-system delete deploy calico-kube-controllers --ignore-not-found=true
 
-# delete legacy nginx daemonset
-$KUBECTL -n kube-system delete daemonset nginx-ingress-controller --ignore-not-found=true
```

Comment:
How do we cleanup the "legacy" ingress controller if we drop this line?

Reply:
I'll cleanup them manually before merging app into appcollections